phpunit.xml.dist

```diff
@@ -12,15 +12,21 @@
          bootstrap="Tests/bootstrap.php"
 >
   <testsuites>
-    <testsuite name="Teamspeak 3 PHP Framework Test Suite">
+    <testsuite name="MyOwnBundle Test Suite">
```

Comment:
Any reason for naming this `MyOwnBundle`? Generic `TeamSpeak3 PHP Framework` package name seems to make more sense here to me.

Reply:
Nevermind, please disregard. I see this is the `.dist` file.